api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+        $ref: '#/definitions/Cursor'
+      events:
+        type: array
+        items:
+          $ref: '#/definitions/Event'
+
+  EventType:
+    description: An event type defines the schema and its runtime properties.
+    properties:
+      name:
+        type: string
+        description: |
+          Name of this EventType. Encodes the owner/responsible for this EventType.
+
+          The name for the EventType SHOULD follow the pattern, but is not enforced
+          'organization:team:owning_application:event-type', for example
+          'zalando:bazaar:price-service:price-change'.
+
+          The components of the name are:
+          * Organization: the organizational unit where the team is located; can be omitted.
+          * Team name: name of the team responsible for owning application; can be omitted.
+          * Owning application: SHOULD match the field owning_application; indicates
+          * EventType name: name of this EventType; SHOULD end in ChangeEvent for DataChangeEvents; MUST be
+          in the past tense for BusinessEvents.
+
+          (TBD: how to deal with organizational changes? Should be reflected on the name of the EventType?
+          Isn t it better to omit [organization:team] completely?)
+      owning_application:
+        type: string
+        description: |
+          Indicator of the Application owning this `EventType`.
+        example: price-service
+      type:
+        type: string
+        enum:
+          - undefined
+          - data
+          - business
+        description: |
+          Defines the type of this EventType.
+
+          The value set will influence, if not set otherwise, the default set of validation-strategies,
+          enrichment-strategies, and the effective_schema in the following way:
+
+          - `undefined`: No predefined changes apply. `effective_schema` is exactly the same as the
+            `EventTypeSchema`.
+            Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
+
+          - `data`: Events of this type will be DataChangeEvents. `effective_schema` contains `metadata`, and
+            adds fields `data_op` and `data_type`. The passed EventTypeSchema defines the schema of `data`.
+            Default validation_strategy for this `EventType` is `[{name: 'datachange-schema-validation'}]`.
+
+          - `business`: Events of this type will be BusinessEvents. `effective_schema` contains `metadata`
+            and any additionally defined properties passed in the `EventTypeSchema` directly on top level of
+            the Event. If name conflicts arise, creation of this EventType will be rejected.
+            Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
+      topic:
+        type: string
+        description: |
+          References the name of the low-level topic that is holding the stream of Events for this EventType.
+          Usually is the same as the name property.
```

Comment:
Both, name and topic are must be unique ids. Why we do not simply go with only topic (or name)?
I don't see the value 2 unique ids -- and find it confusing from API design pov.
I also find it strange, to have the topics as single resources -- it is an id attribute of the event type.
Why not simply and consistently use event-types as root resources for event type management and event creation?